olp-cpp-sdk-core/src/http/winhttp/NetworkWinHttp.cpp

```diff
@@ -977,7 +1002,17 @@ NetworkWinHttp::RequestData::RequestData(
       resumed(false),
       ignore_data(request.GetVerb() == NetworkRequest::HttpVerb::HEAD),
       no_compression(false),
-      uncompress(false) {}
+      uncompress(false),
+      in_use(false) {}
+
+NetworkWinHttp::RequestData::RequestData()
+    : http_request(NULL),
+      request_id(-1),
```

Comment:
```suggestion
      request_id(RequestIdConstants::RequestIdInvalid),
```
by using -1, you will use not 'invalid' value, but `RequestIdMax`, because `RequestID`is unsigned type